arango/batch.py

```diff
@@ -85,14 +101,18 @@ def handle_request(self, request, handler):
         :returns: the batch job or None
         :rtype: arango.batch.BatchJob
         """
-        self._requests.append(request)
-        self._handlers.append(handler)
 
-        if not self._return_result:
-            return None
-        batch_job = BatchJob()
-        self._batch_jobs.append(batch_job)
-        return batch_job
+        to_return = None
```

Comment:
The case i was envisioning was an aggregator- where one thread owns the batch, tracks its size, and submits it every time it gets to a given size.  However, there is currently no way to get the output of a batch if you are that managing process, since the batch contents are deleted after commit.  It also wouldnt be a bad thing to not have to keep track of your commits if you were going to put each of the BatchJobs into a list anyway.